### ISSUE TITLE
3.1.12 - Display entry page of document as default page (Content view)

### ISSUE THREAD

**Issue by janahoffmann** [gbhl/bhl-europe#94]
Show first page of publication, not first page scanned (ideal: with author and title)

Precondition: correct pagination of scanned item in metadata

**Comment by zhengl** [comment]
Only if the pagination is available.

**Comment by lobajuluwa** [comment]
Page types can be handled based on filenames, content viewer will already handle this. Overall the function itself is in place, needs data in OLEF before Issue can be Closed for testing.

**Comment by JiriFrank** [comment]
Done and testable.

**Comment by JiriFrank** [comment]
3.1.12 - Display entry page of document as default page (Content view)

COR number: 3.1.12
Testing platform: http://bhl-test.nhm.ac.uk/portal/
Function: Content viewer

Description:

Show first page of publication, not first page scanned (ideal: with author and title).

The content viewer is set up for this function, but we do not have the information about the title page at the moment. First page of the book is displayed.

---

Testers:
@AnneSch
@AntonioGVH
@fwelter
@grahamhrbge1670
@heimor
@HenningScholz
@JFTester
@JiriFrank
@LarissaS
@RalfH

**Comment by JiriFrank** [comment]
3.1.12 - Display entry page of document as default page (Content view)

COR number: 3.1.12
Testing platform: http://bhl-test.nhm.ac.uk/portal/
Function: Content viewer

Description:

Show first page of publication, not first page scanned (ideal: with author and title).

The content viewer is set up for this function, but we do not have the information about the title page at the moment. First page of the book is displayed.

---

@GregKenicer
@GeoffHarper
@KasiaGoral
@MartinaMetzger
@DanielFisher
@NeilWoodcock
@SaraPerzley
@SaraCarlton
@HannaKoivula
@PaiviLipsanen

**Comment by JiriFrank** [comment]
3.1.12 - Display entry page of document as default page (Content view)

COR number: 3.1.12
Testing platform: http://bhl-test.nhm.ac.uk/portal/
Function: Content viewer

Description:

Show first page of publication, not first page scanned (ideal: with author and title).

The content viewer is set up for this function, but we do not have the information about the title page at the moment. First page of the book is displayed.

---

@PaiviJaakkola
@SiniKarki
@TiinaOnttonen

**Comment by LarissaS** [comment]
For Liebrechts and De Moor- cover is displayed, for Trouessart, I see 1 result in autocompletion but no results then I click on it.
For Aigret - blank page is displayed, than I look in thumbnail view then 4 first pages are blank and then scanned pages start.
For Crampton - no images at all, after 5 minutes text appeared but 4 first pages remained blank. I did not wait longer.

.
-----Original Message-----

**Comment by HenningScholz** [comment]
Clear, we need better data to display the right page.

---

Von: JiriFrank [mailto:reply@reply.github.com]
Gesendet: Do 12.04.2012 19:04
An: Scholz, Henning
Betreff: Re: [bhle] 3.1.12 - Display entry page of document as default page (Content view) (#94)

3.1.12 - Display entry page of document as default page (Content view)

COR number: 3.1.12
Testing platform: http://bhl-test.nhm.ac.uk/portal/
Function: Content viewer

Description:

Show first page of publication, not first page scanned (ideal: with author and title).

The content viewer is set up for this function, but we do not have the information about the title page at the moment. First page of the book is displayed.

---

Testers:
@AnneSch
@AntonioGVH
@fwelter
@grahamhrbge1670
@heimor
@HenningScholz
@JFTester
@JiriFrank
@LarissaS
@RalfH

---

**Comment by RalfH** [comment]
As in 3.1.11: It's all about the data and the informations provided. Not much can be commented at the moment.

-----Ursprüngliche Nachricht-----
Von: JiriFrank [mailto:reply@reply.github.com] 
Gesendet: Donnerstag, 12. April 2012 19:04
An: Hand, Ralf
Betreff: Re: [bhle] 3.1.12 - Display entry page of document as default page (Content view) (#94)

3.1.12 - Display entry page of document as default page (Content view)

COR number: 3.1.12
Testing platform: http://bhl-test.nhm.ac.uk/portal/
Function: Content viewer

Description:

Show first page of publication, not first page scanned (ideal: with author and title).

The content viewer is set up for this function, but we do not have the information about the title page at the moment. First page of the book is displayed.

---

Testers:
@AnneSch
@AntonioGVH
@fwelter
@grahamhrbge1670
@heimor
@HenningScholz
@JFTester
@JiriFrank
@LarissaS
@RalfH

---

**Comment by grahamtestrbge** [comment]
This does not work well, checked 4 titles, and couldn't retrieve Title Page easily, however, this must be because there is no flag in the ToC box to say that a page is a Title Page

**Comment by KasiaGoral** [comment]
Works for 'Vier landw. Studiengänge durch die Gemarkung Niederlahnstein und
ihre Flurdistrikte'.
For 'Über die unterdevonischen Schichten bei Coblenz' the cover page
appears.
For 'The vegetation of Caithness considered in relation to the geology' the
first 3 pages are blank.

**Comment by AntonioGVH** [comment]
I looked for:
"Antwerpsche analytische flora inhoudende de beschyving van alle planten, welke in de provincie Antwerpen, in de vrye natuer, groeijen of er algemeen gekweekt worden. Eerste del"

And the first page (default, was the hard cover of the book (no authors's name neither book title).

Looking through the book I noticed that it jumps from page 192 to page 233 and I don't know if this is due to bad digitization or because plates were not included for any special reason.

I include an screen shot.

Best

Antonio

-----Mensaje original-----
De: JiriFrank [mailto:reply@reply.github.com] 
Enviado el: jueves, 12 de abril de 2012 19:04
Para: AntonioGVH
Asunto: Re: [bhle] 3.1.12 - Display entry page of document as default page (Content view) (#94)

3.1.12 - Display entry page of document as default page (Content view)

COR number: 3.1.12
Testing platform: http://bhl-test.nhm.ac.uk/portal/
Function: Content viewer

Description:

Show first page of publication, not first page scanned (ideal: with author and title).

The content viewer is set up for this function, but we do not have the information about the title page at the moment. First page of the book is displayed.

---

Testers:
@AnneSch
@AntonioGVH
@fwelter
@grahamhrbge1670
@heimor
@HenningScholz
@JFTester
@JiriFrank
@LarissaS
@RalfH

---
